plugins/base/src/test/resources/expect/signatureTest/out/images/logo-text.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="83" height="27" viewBox="0 0 83 27" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Are you sure this is signature specific file? If it does not, it should definitely be moved out of this PR